src/users_db.rs

```diff
@@ -264,7 +285,14 @@ impl UsersDb {
                 email       TEXT NOT NULL UNIQUE,
                 password    TEXT NOT NULL,
                 secret      TEXT NOT NULL,
-                is_admin    BOOL NOT NULL DEFAULT 0
+                is_admin    BOOL NOT NULL DEFAULT 0,
+                push_uri    TEXT,
+                push_key    TEXT
```

Comment:
Good point, I had only considered the single client. It made sense to put it in when it was so simple but yeah, I'll move it out when I next update the PR. I believe I will still need the auth middleware changes so I can know what user is identified by the bearer token.